opm/input/eclipse/EclipseState/Grid/EclipseGrid.cpp

```diff
@@ -2176,3 +2344,23 @@ std::vector<double> EclipseGrid::createDVector(const std::array<int,3>& dims, st
 
 
 
+namespace Opm {
+    EclipseGridLGR::EclipseGridLGR(const std::string& self_label, const std::string& father_label_, 
+                                   int father_lgr_level, std::size_t nx, std::size_t ny, 
+                                   std::size_t nz, vec_size_t father_lgr_index)
+    : EclipseGrid(nx,ny,nz), father_label(father_label_), father_global(father_lgr_index)
+    {
+        init_father_global();
+        lgr_label= self_label;
+        lgr_level = father_lgr_level + 1 ;   
+    }
+    void EclipseGridLGR::init_father_global()
+    {
+        std::sort(father_global.begin(),father_global.end(), std::less<std::size_t>());
```

Comment:
Note that `std::less<>` is the default comparison operator for `std::sort()` so there's usually no need to explicitly pass `std::less<>` as a comparison function.